shared/tracker/bio.render.desktop.js

```diff
@@ -16,45 +16,51 @@ import type {BioProps} from './bio.render'
 export default class BioRender extends Component {
   props: BioProps;
 
-  onClickAvatar () {
+  _onClickAvatar () {
```

Comment:
i was hoping flow would notice these are private and notice we're calling a check on the userInfo being null but it didn't. maybe we have to turn something on. just leaving this. we should likely adopt this everywhere. i know @MarcoPolo does this sometimes already